src/main/java/com/skynest/tests/utils/BaseTransformer.java

```diff
@@ -0,0 +1,14 @@
+package com.skynest.tests.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class BaseTransformer {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This private constant should follow the naming convention used elsewhere in the project